src/scenes/Editor/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import AceEditor from 'react-ace';
+import emailjs from 'emailjs-com';
+import './style.scss';
+
+import 'ace-builds/src-noconflict/mode-html';
+import 'ace-builds/src-noconflict/theme-monokai';
+
+export default class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const starterCode = `<html>
+  <head>
```

Comment:
Ahhhhh tab this over please

Reply:
Isn't tabbed because the `` is literal and would reflect extra indenting in the editor